faunadb-scala/src/test/scala/faunadb/ClientSpec.scala

```diff
@@ -680,14 +704,15 @@ class ClientSpec
     val concat2R = client.query(Concat(Arr("Magic", "Missile"), " ")).futureValue
     concat2R.to[String].get shouldBe "Magic Missile"
 
-    // Contains
-    val containsR = client.query(Contains("favorites" / "foods", Obj("favorites" -> Obj("foods" -> Arr("crunchings", "munchings"))))).futureValue
-    containsR.to[Boolean].get shouldBe true
-
-    client.query(Contains("field", Obj("field" -> "value"))).futureValue shouldBe TrueV
-    client.query(Contains(1, Arr("value0", "value1", "value2"))).futureValue shouldBe TrueV
-
-    client.query(Contains("a" / "nested" / 0 / "path", Obj("a" -> Obj("nested" -> Arr(Obj("path" -> "value")))))).futureValue shouldBe TrueV
+    // NOT AVAILABLE WITH API V4
```

Comment:
my suggestion is to revert this and leave it breaking, we gonna bring back `Contains` function because we did create a deprecation plan for the function